'Process' object has no attribute 'exception'